src/common/low_precision_transformations/src/recurrent_cell.cpp

```diff
@@ -0,0 +1,237 @@
+        OutputVector{
+            fq_R, dequantization_R, dequantization_without_subtract_R
+        });
+
+    const auto lstm_seq = ngraph::pattern::wrap_type<ngraph::opset5::LSTMSequence>(
+        {X_in, H_in, C, S, W_in, R_in, B});
+    const auto gru_seq  = ngraph::pattern::wrap_type<ngraph::opset5::GRUSequence>(
+        {X_in, H_in,    S, W_in, R_in, B});
+
+    ngraph::graph_rewrite_callback callback = [this](pattern::Matcher& m) {
+        auto op = m.get_match_root();
+        if (transformation_callback(op)) {
+            return false;
+        }
+
+        return transform(*context, m);
+    };
+
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        std::make_shared<pattern::op::Or>(
+            OutputVector {
+                lstm_seq,
+                gru_seq
+            }),
+        "RecurrentCellTransformation");
+
+    this->register_matcher(m, callback);
+}
+
+bool RecurrentCellTransformation::transform(TransformationContext& context, ngraph::pattern::Matcher& m) {
+    const auto lstm = m.get_match_root();
+    if (!canBeTransformed(context, lstm)) {
+        return false;
+    }
+    for (size_t parentIndex = 0ul; parentIndex < lstm->get_input_size(); parentIndex++) {
+        auto lstm_parent = lstm->get_input_node_shared_ptr(parentIndex);
+        if (is_type<ngraph::opset1::FakeQuantize>(lstm_parent)) {
+            auto fq_parent = lstm_parent->get_input_node_shared_ptr(0);
+            if (is_type<ngraph::opset5::Constant>(fq_parent)) {
+                auto fq_node = as_type_ptr<ngraph::opset1::FakeQuantize>(lstm_parent);
+                const QuantizationDetails quantizationDetails = QuantizationDetails::getDetails(fq_node);
+                const auto precisionsAttribute = getAttributeFromOutput<PrecisionsAttribute>(lstm_parent);
+                const auto precisions = precisionsAttribute.empty()
+                                            ? defaultPrecisions
+                                            : precisionsAttribute.as<PrecisionsAttribute>().value();
+                const DataPrecision dataPrecision = getDataPrecision(lstm_parent, quantizationDetails, precisions);
+                auto QDQ = NetworkHelper::decomposeFakeQuantize(fq_node,
+                                                                  dataPrecision.precision,
+                                                                  dataPrecision.min,
+                                                                  dataPrecision.max,
+                                                                  dataPrecision.hasZeroPoint,
+                                                                  updatePrecisions);
+                std::shared_ptr<ngraph::Node> new_fq = std::get<0>(QDQ);
+                std::shared_ptr<ngraph::Node> deq_multiply = std::get<1>(QDQ);
+                auto multiply_parent = deq_multiply->get_input_node_shared_ptr(0);
+                ov::disable_constant_folding(multiply_parent);
```

Comment:
Actually I don't remember the exact reason I disabled it for multiply parent.
Something was not working.
What would be best option for the first op after constant then?

Reply:
I suppose we need to pass to the  `disable_constant_folding` as a parameter `multiply_parent` if it isn't Subtract, or `multiply_parent->get_input_node_shared_ptr(0)` otherwise

Reply:
Done. Please double check